resources/js/features/home/components/+root/FrontPageNews/NewsCard.tsx

```diff
@@ -1,49 +1,59 @@
-import { type FC, type ReactNode, useState } from 'react';
+import dayjs from 'dayjs';
+import { type FC, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-
+import { LuPin } from 'react-icons/lu';
+
+import { BaseBadge } from '@/common/components/+vendor/BaseBadge';
+import {
+  BaseTooltip,
+  BaseTooltipContent,
+  BaseTooltipPortal,
+  BaseTooltipTrigger,
+} from '@/common/components/+vendor/BaseTooltip';
+import { DiffTimestamp } from '@/common/components/DiffTimestamp';
 import { usePageProps } from '@/common/hooks/usePageProps';
 import { cn } from '@/common/utils/cn';
+import { formatDate } from '@/common/utils/l10n/formatDate';
 
 interface NewsCardProps {
-  authorDisplayName: string;
-  lead: string;
-  PostedAt: ReactNode;
-  title: string;
+  news: App.Data.News;
 
   className?: string;
-  href?: string;
-  imageSrc?: string;
   tagLabel?: string;
 }
 
-export const NewsCard: FC<NewsCardProps> = ({
-  authorDisplayName,
-  href,
-  imageSrc,
-  lead,
-  PostedAt,
-  title,
-  className,
-}) => {
+export const NewsCard: FC<NewsCardProps> = ({ news, className }) => {
   const { ziggy } = usePageProps();
 
   const { t } = useTranslation();
 
+  const isRecentPost = dayjs.utc().diff(dayjs.utc(news.createdAt), 'hour') < 24;
+
   return (
     <a
-      href={href}
+      href={news.link ?? '#'}
       className={cn(
-        'group -mx-2 cursor-pointer gap-6 rounded-xl p-2 sm:flex',
+        'group -mx-2 cursor-pointer gap-6 rounded-xl p-1.5 sm:flex',
         'hover:bg-neutral-950/30 hover:light:bg-neutral-100',
+        'border-2 border-transparent',
 
         ziggy.device === 'desktop' ? 'sm:bg-embed' : '',
         ziggy.device === 'mobile' ? 'bg-embed' : '',
 
+        news.pinnedAt ? 'border-rose-600' : '',
```

Comment:
I know the red is meant to draw the eye, but it makes me think there's a problem (which may be true for something that's pinned). Maybe it could be orange (warning) or just something with high--contrast instead?

Reply:
Updated the color in latest:
![Screenshot 2024-12-27 at 9 47 48 AM](https://github.com/user-attachments/assets/9028fd91-cde1-4853-8919-d49d8b4e89e6)